### ISSUE TITLE
Freezes under Windows 10 Pro

### ISSUE THREAD

**Issue by ghost** [fritzing/fritzing-app#3304]
I give up.  Even tried compatibility mode.

Installed in Windows 10 Pro and as soon as the UI loads it freezes solid.

**Comment by dcconn** [comment]
This happens to me also.

**Comment by dcconn** [comment]
I think I know what the problem was - I found a modal window hidden behind the main window asking about removing files prior to update. This windows required an answer. I was using task view and saw it there I've included a screenshot of the modal window after I clicked OK.
![fritzing](https://cloud.githubusercontent.com/assets/8344519/22302428/b2577ef8-e326-11e6-9f72-c523502432f2.png)

Once I allowed update to finish, I restarted fritzing and it seems ok :).

Regards,
David.
